ably/state.go

```diff
@@ -29,7 +29,7 @@ func (st StateType) String() string {
 	case StateChan:
 		return "channel"
 	default:
-		return ""
+		return "invalid"
```

Comment:
Using empty string will hide bugs, user should never see invalid when he/she prints state received from the `ably.Client`. Seeing "invalid" could help troubleshooting. My original version was just wrong.